app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: str,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        washed_cars = [car for car in cars
+                       if car.clean_mark < self.clean_power]
+        income = sum(self.calculate_washing_price(car) for car in washed_cars)
+        for car in washed_cars:
+            self.wash_single_car(car)
+        return round(income, 1)
```

Comment:
it's redundant to use both the `for` loop and comprehension here. Your code spends extra time on it.